abstract_test.go

```diff
@@ -443,17 +438,17 @@ func TestResolveTypeOnInterfaceYieldsUsefulError(t *testing.T) {
 				nil,
 			},
 		},
-		Errors: []graphqlerrors.GraphQLFormattedError{
-			graphqlerrors.GraphQLFormattedError{
+		Errors: []FormattedError{
```

Comment:
@chris-ramon there are many things to improve but this pull request is the big one. 
let's merge it, and refactor the little things later..